src/components/home/Upcoming.jsx

```diff
@@ -1,5 +1,17 @@
+import UpcomingCard from "./UpcomingCard";
+
 const Upcoming = () => {
-  return <div>Upcoming</div>;
+  return (
+    <div>
+      Upcoming
+      <div className="m-11 max-w-4xl grid grid-cols-2 gap-4 ">
+        <UpcomingCard></UpcomingCard>
```

Comment:
```suggestion
        <UpcomingCard />
```

repeat for other upcoming cards